src/jit/gentree.h

```diff
@@ -1080,6 +1080,17 @@ struct GenTree
     bool OperIsInitBlkOp() const;
     bool OperIsDynBlkOp();
 
+    static
+    bool OperIsBlk(genTreeOps gtOper)
+    {
+        return (gtOper == GT_OBJ);
+    }
+
+    bool OperIsBlk()
+    {
```

Comment:
Should these be named OperIsObj? It's not GT_BLK after all.

---
In reply to: [75344836](https://github.com/dotnet/coreclr/pull/6780#discussion_r75344836) [](ancestors = 75344836)